Source/Krypton Components/Krypton.Ribbon/Group Contents/KryptonRibbonGroupTrackBar.cs

```diff
@@ -434,10 +434,10 @@ public int Value
         [Category(@"Behavior")]
         [Description(@"Change to apply when a small change occurs.")]
         [DefaultValue(1)]
-        public int SmallChange
+        public int? SmallChange
```

Comment:
This value will never be null, so do not make it nullable

Reply:
Added `[DisallowNull]` flag